crypto/keys/keybase.go

```diff
@@ -0,0 +1,364 @@
+package keys
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	dbm "github.com/tendermint/tmlibs/db"
+	tcrypto "github.com/tendermint/tendermint/crypto"
+
+	"github.com/cosmos/cosmos-sdk/crypto"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/bip39"
+	"github.com/cosmos/cosmos-sdk/crypto/keys/hd"
+)
+
+var _ Keybase = dbKeybase{}
+
+// Language is a language to create the BIP 39 mnemonic in.
+// Currently, only english is supported though.
+// Find a list of all supported languages in the BIP 39 spec (word lists).
+type Language int
+
+const (
+	// English is the default language to create a mnemonic.
+	// It is the only supported language by this package.
+	English Language = iota + 1
```

Comment:
Multiple versions of English should be added imo. There are many different wordlists that would be sensible to support. This can be changed later, so doesn't block this. If we don't have to worry about cross compatibility, then we can just comment out all the languages we don't yet support and fix them later.

Doesn't block this PR.

Reply:
Commented in #1411